src/components/Login/index.tsx

```diff
@@ -16,15 +16,47 @@ import MainLogoSvg from "@/assets/svg/CourtCanva-main-LOGO.svg";
 import { IconContext } from "react-icons";
 import { FaEnvelope } from "react-icons/fa";
 import { FcGoogle } from "react-icons/fc";
+import { useGoogleLogin } from "@react-oauth/google";
 import React from "react";
+import { api } from "../../utils/axios";
+import { useDispatch } from "react-redux";
+import { useStoreSelector } from "@/store/hooks";
+import { updateUserInfo } from "@/store/reducer/userSlice";
 
 interface Props {
   isOpen: boolean;
   onClose: () => void;
+  updateLoginData: (data: any) => void;
 }
 
 function LoginModalContent(props: Props) {
   const initialRef = React.useRef(null);
+  const dispatch = useDispatch();
+  // Send request to backend after the request from front-end has been approved by Google
+  /* istanbul ignore next */
+  const handleSuccess = (codeResponse: any) => {
+    api(process.env.NEXT_PUBLIC_API_BASE_URI + "/auth/google"!, {
```

Comment:
You have baseUrl in axiosInstance
<img width="774" alt="image" src="https://user-images.githubusercontent.com/61776449/176455444-e6c7bea4-5f84-4529-a810-0ba622f76a83.png">